running_steps.md

```diff
@@ -0,0 +1,14 @@
+The basic steps to run the PHANGS version 2 script. The PHANGS_v2 scripts processing can be divided into 4 steps: staging, imaging, processing and derving the data products. The detailed description for these 4 steps is in README.md. Here is the detailed description of steps to use PHANGS v2 scripts to product images.  
+- Generate the key folder by running `python start_project.py`
+    - You will go through bunch of questions which are quite straightforward. 
+    - You will also specify the location of [keys/](keys) directory in which you have bunch of files containning the parameter settings for staging and imaging
```

Comment:
Do you make this directory or the files inside or does something else do it for you? Is this part of the questions you go through?

"containning" -> "containing"